functions/go/render-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -135,12 +133,32 @@ func (p *HelmChartInflationGeneratorPlugin) runHelmCommand(
 	return stdout.Bytes(), err
 }
 
-// createNewMergedValuesFile replaces/merges original values file with ValuesInline.
-func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFile() (
-	path string, err error) {
+// createNewMergedValuesFiles replaces/merges original values file with ValuesInline.
+func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFiles(path string) (
+	string, error) {
+	pValues, err := ioutil.ReadFile(path)
+	if err != nil {
+		if u, urlErr := url.Parse(path); urlErr == nil {
+			if (u.Scheme == "http" || u.Scheme == "https") {
+				resp, err := http.Get(path)
+				if err != nil {
+					return "", err
+				}
+				defer resp.Body.Close()
+				pValues, err = ioutil.ReadAll(resp.Body)
+				if err != nil {
+					return "", err
+				}
+			} else { // url scheme is not http or https
+				return "", fmt.Errorf("unsupported url type: %s", path)
```

Comment:
To be more accurate, let's use `unsupported URL scheme`.

Reply:
done